File: Modul 3/PRAK304_2410817310016_ARIFFATAHILLAHPERMANA.py
x = int(input())
if x == 0:
    print("Nol")
elif x >= 1 and x <= 9:
    print("Satuan")
elif x >= 11 and x <= 19:
    print("Belasan")
elif (x >= 20 and x <= 99) or x == 10:
    print("Puluhan")
else:
    print("Anda Menginput Melebihi Limit Bilangan")

File: Modul 3/PRAK304_2410817310016_ARIFFATAHILLAHPERMANA.c
#include <stdio.h>

int main() {
    int x;
    scanf("%d", &x);
    
    if (x == 0) {
        printf("Nol");
    } else if (x >= 1 && x <= 9) {
        printf("Satuan");
    } else if (x >= 11 && x <= 19) {
        printf("Belasan");
    } else if ((x >= 20 && x <= 99) || x == 10) {
        printf("Puluhan");
    } else {
        printf("Anda Menginput Melebihi Limit Bilangan");
    }
    
    return 0;
}